core/src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -55,49 +74,81 @@
     /**
      * Constructs a new term query.
      *
-     * @param name  The name of the field
-     * @param value The value of the term
+     * @param fieldName  The name of the field
+     * @param text The value of the term
      */
-    public FuzzyQueryBuilder(String name, Object value) {
-        this.name = name;
-        this.value = value;
+    public FuzzyQueryBuilder(String fieldName, Object text) {
+        this.fieldName = fieldName;
+        this.text = text;
     }
 
-    /**
-     * Sets the boost for this query.  Documents matching this query will (in addition to the normal
-     * weightings) have their score multiplied by the boost provided.
-     */
-    @Override
-    public FuzzyQueryBuilder boost(float boost) {
-        this.boost = boost;
-        return this;
+    public String fieldName() {
+        return this.fieldName;
+    }
+
+    public Object text() {
+        return this.text;
     }
 
     public FuzzyQueryBuilder fuzziness(Fuzziness fuzziness) {
-        this.fuzziness = fuzziness;
+        this.fuzziness = (fuzziness == null) ? DEFAULT_FUZZINESS : fuzziness;
         return this;
     }
+    
+    public Fuzziness fuzziness() {
+        return this.fuzziness;
+    }
 
     public FuzzyQueryBuilder prefixLength(int prefixLength) {
         this.prefixLength = prefixLength;
         return this;
     }
+    
+    public int prefixLength() {
+        return this.prefixLength;
+    }
 
     public FuzzyQueryBuilder maxExpansions(int maxExpansions) {
         this.maxExpansions = maxExpansions;
         return this;
     }
 
+    public int maxExpansions() {
+        return this.maxExpansions;
+    }
+
     public FuzzyQueryBuilder transpositions(boolean transpositions) {
       this.transpositions = transpositions;
       return this;
     }
 
+    public boolean transpositions() {
+        return this.transpositions;
+    }
+
     public FuzzyQueryBuilder rewrite(String rewrite) {
         this.rewrite = rewrite;
         return this;
     }
 
+    public String rewrite() {
+        return this.rewrite;
+    }
+
+    /**
+     * Sets the boost for this query.  Documents matching this query will (in addition to the normal
+     * weightings) have their score multiplied by the boost provided.
+     */
+    @Override
```

Comment:
I think we can simply re-use the comment of the super method by leaving this one undocumented?